index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+    <link rel="stylesheet" href="styles.css">
```

Comment:
- [ ] To place the metadata like stylesheet links I suggest you use `<head>` element

> The` <head>` element is a container for metadata (data about data) and is placed between the `<html>` tag and the `<body>` tag. Metadata is data about the `HTML` document. Metadata is not displayed. Metadata typically define the document` title`, `character set`, `styles`,` scripts`, and other` meta` information.